src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -155,8 +155,8 @@
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Unix\libc\Interop.close.cs">
-      <Link>Interop\Unix\libc\Interop.close.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.Close.cs">
+      <Link>Interop\Unix\libc\Interop.Close.cs</Link>
```

Comment:
I left libc in the link path. I'm not going to bother resetting CI since all PRs are blocked on this right now. 

Most projects don't have the library name in the link path, but some do. Let's decide which way we want to standardize and I'll fix up the whole lot to match. cc @stephentoub